### DISCUSSION TITLE
Sandboxie Plus  - Portable (appTitle)

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2488] [Q&A Feedback] [answered]
I have been using Sandboxie Plus as portable since the very beginning. Once the application is launched the window bar says: "Sandboxie-Plus v1.5.3". If I remember correctly it used to say "Sandboxie-Plus v1.5.3 - Portable". Was the Portable part removed from the window bar in one of the recent updates or my Sandboxie installation got corrupted?

**Comment by offhub** [comment, marked as answer]
[Removed with v1.3.0 / 5.58.0](https://github.com/sandboxie-plus/Sandboxie/blob/51984fe799d6e3d914c96a413079022df12c2d1b/SandboxiePlus/SandMan/SandMan.cpp#L1354)
